keylime/cert_utils.py

```diff
@@ -24,6 +25,9 @@
 # when parsing them with python-cryptography fails, and in this case, we
 # try to read the parsed certificate again into python-cryptograhy.
 
+OID_HW_MODULE_NAME = "1.3.6.1.5.5.7.8.4"
+OID_HWTYPE_TPM = "2.23.133.1.2"
```

Comment:
From where this comes from?

Reply:
This is from the certificate fields section of the [TPM 2.0 Keys for device ID spec](https://trustedcomputinggroup.org/wp-content/uploads/TPM-2p0-Keys-for-Device-Identity-and-Attestation_v1_r12_pub10082021.pdf).

You can find it in section 8.1, the hardwareModuleName part of the SubjectAltName on page 57.

Reply:
Please add comments pointing to the origin of these OIDs.

Reply:
I have added this in the newest commit